app/presenters/case_studies_presenter.rb

```diff
@@ -0,0 +1,18 @@
+class CaseStudiesPresenter < ContentItemPresenter
+  attr_reader :content_item, :view_context
+
+  def initialize(content_item, view_context)
+    super(content_item)
+    @content_item = content_item
+    @view_context = view_context
+  end
+
+  def title_and_context
+    {
+      title: content_item.title,
+      context: I18n.t("formats.#{content_item.document_type}", count: 1),
+      context_locale: view_context.t_locale_fallback("formats.#{content_item.document_type}", count: 1),
+      average_title_length: "long",
+    }
+  end
+end
```

Comment:
It might be worth just putting this straight into the view (take a look at the take_part controller), which would allow you to cut out the entire presenter.